### DISCUSSION TITLE
Formatting using wasm compiler

### DISCUSSION THREAD

**Discussion by hatemhosny** [gleam-lang/gleam#3029] [General]
Congratulations for the great work. 

Is it possible to use the wasm compiler to format gleam code provided as a string?
If yes, I would appreciate some guidance (if possible with some js code sample)

Thank you.

**Comment by lpil** [comment]
Not today I'm afraid.

**Comment by hatemhosny** [comment]
Thank you for the reply.
please see https://github.com/gleam-lang/gleam/discussions/3041 for context.
